common/oidc-client/oauth2-proxy/components/allow-unauthenticated-issuer-discovery/README.md

```diff
@@ -0,0 +1,45 @@
+# Unauthenticated issuer discovery
+
+If you're running Kubernetes with kind, vCluster, minikube or some other
+tool for local development on Kubernetes, there is a high change that the
```

Comment:
then there is a high chance that the Kubernetes OIDC Issuer is managed within the cluster behind self-signed
certificates.